lib/galaxy/datatypes/binary.py

```diff
@@ -331,22 +331,67 @@ class GenericAsn1Binary(Binary):
     edam_data = "data_0849"
 
 
-class BamNative(CompressedArchive):
+class _BamOrSam():
```

Comment:
```suggestion
class _BamOrSam:
```
?

Reply:
Odd that the linter did not complain. 

One thing that I was also wondering is why I can't specify the `MetadataElement`s in this class, but need to duplicate them in the sam and bam classes?